code/game/objects/items/weapons/nt_melee.dm

```diff
@@ -0,0 +1,118 @@
+//Warning! If you change icon_state or item_state, make sure you change path for sneath as well. icons/obj/sneath.dmi
+/obj/item/weapon/tool/sword/nt
+	name = "NT Shortsword"
+	desc = "A saint looking sword, made to do God's work."
+	icon = 'icons/obj/nt_melee.dmi'
+	icon_state = "nt_shortsword"
+	item_state = "nt_shortsword"
+	force = WEAPON_FORCE_DANGEROUS
+	throwforce = WEAPON_FORCE_WEAK
+	armor_penetration = ARMOR_PEN_DEEP
+	spawn_blacklisted = TRUE
+	aspects = list(SANCTIFIED)
+	price_tag = 750
```

Comment:
Knock this down to 300$, tops. Its components cost roughly 260$ to make, and it can be mass-produced.
```suggestion
	price_tag = 300
```

Reply:
Done